### ISSUE TITLE
API: list pd members

### ISSUE THREAD

**Issue by siddontang** [pingcap/pd#236]
like etcd get members API, but we should do it for ourselves. 

```
GET http://127.0.0.1:9090/api/v1/members

members = [
    {
       "name"
       "client-urls"
       "peer-urls"
    }
]
```

**Comment by siddontang** [comment]
@overvenus 

we can add a filed `is-leader: true/false` maybe.

**Comment by overvenus** [comment]
what about #237 ?

`is-leader` pd or etcd?

**Comment by siddontang** [comment]
em, no need, ignore.

**Comment by overvenus** [comment]
closed via #238
